tests/associations/belongs-to-many.test.js

```diff
@@ -67,10 +85,15 @@ describe('Associations(BelongsToMany)', function() {
       }),
       test.models.Hobby.create({
         name: 'Querty'
+      }),
+      test.models.Thing.create({
+        name: 'Abc'
       })
-    ]).spread(function(p1, p2, h1, h2) {
+    ]).spread(function(p1, p2, h1, h2, t1) {
       return p1.setHobbies([h1,h2]).then(function() {
```

Comment:
just a reminder here, but this can be rewritten more easily (and without whats starting to look like callback hell) as:
```
return Promise.all([
  p1.setHobbies([h1,h2]),
  p2.setHobbies([h2]),
  t1.setPeople([p1])
]);
```